core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7671,6 +7686,10 @@ public static final class Name {
     public static final String CONF_DIR = "alluxio.conf.dir";
     public static final String CONF_DYNAMIC_UPDATE_ENABLED =
         "alluxio.conf.dynamic.update.enabled";
+    public static final String CONF_SYNC_HEARTBEAT_ENABLED =
+        "alluxio.conf.sync.heartbeat.enabled";
+    public static final String CONF_SYNC_HEARTBEAT_INTERVAL_MS =
+        "alluxio.conf.sync.heartbeat.interval.ms";
```

Comment:
```suggestion
        "alluxio.conf.sync.heartbeat.interval";
```
You may give this an alias `alluxio.conf.sync.heartbeat.interval.ms`. I'm fine with that.